### PR TITLE
Add minetest.get_server_uptime() function to Lua API

### PR DESCRIPTION
**Pull request by Bremaweb**

### CODE REVIEW

**Comment by est31** [comment]
:+1:

File: src/server.h [hunk 1]
```diff
@@ -216,6 +216,10 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 
 	// Connection must be locked when called
 	std::wstring getStatusString();
+	double getUptime()
```
Comment (nerzhul) [inline src/server.h]: This function should be const

Comment (nerzhul) [inline src/server.h]: This function should be const

**Comment by nerzhul** [comment]
Except the constness issue it's okay for me. I add the milestone tag.
When you fix it you have my approval

**Comment by sfan5** [comment]
:+1: if you fix what @nerzhul requested.

**Pull request merged**